RobotoFonts/Remotion.ThirdPartyDependency.RobotoFonts.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="..\_._">
+      <Pack>true</Pack>
+      <PackagePath>\</PackagePath>
+    </None>
+  </ItemGroup>
+
+  <PropertyGroup>
+    <Owner>RUBICON IT GmbH</Owner>
+    <Version>2.137</Version>
+    <Title>Roboto Fonts</Title>
+    <Description> Roboto has a dual nature. It has a mechanical skeleton and the forms are largely geometric. At the same time, the font features friendly and open curves. While some grotesks distort their letterforms to force a rigid rhythm, Roboto doesn’t compromise, allowing letters to be settled into their natural width. This makes for a more natural reading rhythm more commonly found in humanist and serif types.
+
+      This is the regular family, which can be used alongside the Roboto Condensed family and the Roboto Slab family.
+
+      To contribute, see github.com/google/roboto </Description>
+    <Authors>Christian Robertson, Google</Authors>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/googlefonts/roboto</RepositoryUrl>
```

Comment:
found the commit hash
https://github.com/googlefonts/roboto/tree/78764ede86ddcdedb42eb9d63c3b24bd8035bb3e

Reply:
I thought we didn't want them as it is not standard to include them? Maybe I misunderstood.

Reply:
I think, yeah, it was a misunderstanding somewhere along the way.